mne_bids/path.py

```diff
@@ -145,6 +159,12 @@ def __ne__(self, other):
         """Compare str representations."""
         return str(self) != str(other)
 
+    def get_fpath(self):
+        """Get the full filepath for this BIDS file."""
+        if self.root:
+            return op.join(self.root, self.basename)
+        return self.basename
```

Comment:
Can this be a property instead? For more Pythonic code, and more consistency with `.basename` (which is not called `get_basename()`)